src/packages/block/block/modals/block-catalogue/block-catalogue-modal.element.ts

```diff
@@ -57,14 +82,9 @@ export class UmbBlockCatalogueModalElement extends UmbModalBaseElement<
 		this._openClipboard = this.data.openClipboard ?? false;
 		this._blocks = this.data.blocks ?? [];
 		this._blockGroups = this.data.blockGroups ?? [];
-	}
 
-	/*
-	#onClickBlock(contentElementTypeKey: string) {
-		this.modalContext?.updateValue({ key: contentElementTypeKey });
-		this.modalContext?.submit();
+		this.#itemManager.setUniques(this._blocks.map((x) => x.contentElementTypeKey));
 	}
-	*/
 
 	render() {
```

Comment:
Create that you tested for broken group keys, could we append a bit of space above such, otherwise really great to make sure that they are shown.

![image](https://github.com/umbraco/Umbraco.CMS.Backoffice/assets/6791648/1e2987d0-85a6-406e-a19c-b44092f0a9b2)